.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
probably good to have something here imo

```suggestion
{
  "printWidth": 80,
  "tabWidth": 2,
  "useTabs": false,
  "semi": true,
  "singleQuote": true,
  "trailingComma": "es5",
  "bracketSpacing": true,
  "jsxBracketSameLine": false,
  "arrowParens": "always",
  "endOfLine": "lf"
}
```

Reply:
up for whatever your preferences are, I included singleQuote and trailingComma from the above as the rest of those are the default settings!